src/Compilers/VisualBasic/Portable/BoundTree/BoundFieldAccess.vb

```diff
@@ -51,7 +51,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
 
 #If DEBUG Then
-                ValidateConstantValue(Me.Type, result)
```

Comment:
Can you elaborate on why we don't need a similar change in `BoundFieldAccess.vb` for the use of this function?